style.css

```diff
@@ -55,13 +53,15 @@ html, body {
 }
 .logo {
     text-decoration: none;
-    flex: 0 0 60px;
-    position: relative;
     z-index: 4;
 }
+.logo__text {
+    margin: 0 5px 10px 10px;
+    font-family: var(--logo-font);
```

Comment:
Why variable for `font-family` instead of regular string?

Reply:
Egija advised that

Reply:
It makes sense for colors, not for fonts

Reply:
I disagree, you should have same approach for all components to be more consistant with thing like fon, font sizes and colors